change/beachball-a311609d-38f8-420d-8cd7-a5a98e5876ba.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Should be minor since it's a new API, in some sense
```suggestion
  "type": "minor",
```